.github/workflows/r-unit-tests.yaml

```diff
@@ -0,0 +1,28 @@
+name: Run R Unit Tests
```

Comment:
Its unusual to have an action for this as they are already being checked in the Rmd check. More usually people would just do this with code coverage